### DISCUSSION TITLE
نصب نسخه قدیمی پنل

### DISCUSSION THREAD

**Discussion by mlm1102003** [Gozargah/Marzban#830] [General]
سلام 
برای نصب نسخه قدیمی تر مثلا  v0.4.0
چه کدی باید بزنیم

**Comment by arian24b** [comment]
بعد از این که مرزبان رو نصب کردین، فایل https://github.com/Gozargah/Marzban/blob/master/docker-compose.yml باز کن و اون قسمتی که نوشته latest به نسخه ای که میخواین تغییر بدین:
`    image: gozargah/marzban:v0.4.0`
بعد از اون:
`docker compose down ;docker compose up -d`
